understanding/22/target-size-minimum.html

```diff
@@ -77,13 +77,21 @@ <h2>Intent of Target Size (Minimum)</h2>
+            <img src="img/pointer-target-example5-revised.png" alt="Three examples of a target adjacent to, or nested within another target (see legend below)" />
+          <figcaption>Three examples of a person's profile image, which is supposed to be a target leading to another page, and a magnifying glass icon, which is supposed to open a lightbox view of the profile image. In the first example, the magnifying icon is outside the picture Its horizontal dimension is 24px. The vertical dimension is below 24px but since there are no adjacent targets above and belowm, the target meets the Success Criterion. In the second example, the icon overlaps with the top-left corner of the image, The image, itself a target, is immediately adjacent to the loupe icon. Measuring from the top of the loupe icon to the nearest point of the image target shows that that the offset is only 20px. The second example therefore fails the Success Criterion. In the third example, the loupe icon sits inside the linked image. Since it is surrounded by a larger target, the icon target fails - the offset to the surrounding and immediately adjacent image target is only 20px at top and bottom.</figcaption>
+      </figure>
+      
+        <figure id="pointer=target-inline-links">
+            <img src="img/pointer-target-example6-inline-links.png" alt="Inline links in a block of text and in a bulleted list (see legend below)" />
+          <figcaption>Two examples show inline text links The frst example is various links in a block of text. Font size is 14px, line height is 16.8px. Depending on the position of links in the text, the vertical offset of links is below 24px. For linked footnote numbers, the horizontal offset is also below 24px. The inline exception allows inline targets to have an offset below the requirement of 24 x 24px.</figcaption>
       </figure>
       
+        <figure id="pointer=target-stacked-links">
+            <img src="img/pointer-target-example7-stacked-links.png" alt="A  bulleted list of stacked links with sufficient and insufficient offset (see legend below)" />
+          <figcaption>The two examples show lists of stacked links as used in navigation, for example in a list of contents of a longer text page. The font size is in both cases 14px. In the first example the line height is 25.8px, exceeding the requirement of 24px. In the second example, the stacked links have a line height of just 16.8px. They do not meet the vertical offset requirement of 24px.</figcaption>
```

Comment:
"The font size is in both cases 14px" sounds a bit awkward. Perhaps smoother "In both cases, the font size is 14px"